numpy/random/tests/test_random.py

```diff
@@ -929,6 +929,52 @@ def test_zipf(self):
                             [3, 13]])
         assert_array_equal(actual, desired)
 
+    def test_complex_normal(self):
```

Comment:
This test seems rather restricted. I like hashing rather long sequences to check for repeatability.